static/gsAdmin/views/customerDetails.spec.tsx

```diff
@@ -4494,6 +4494,262 @@ describe('Customer Details', function () {
       });
     });
   });
+
+  describe('delete billing metric history', function () {
```

Comment:
nit: can we move these tests to the file's own test suite (ie. `deleteBillingMetricHistory.spec.tsx`)? we should try to keep `customerDetails` tests to things specific to `customerDetails` (ie. testing that the action is available to orgs with x feature) and move any tests involving action modals to their own files (ie. testing that you can submit from the action modal and it calls x api)